CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ Mapbox welcomes participation and contributions from everyone.
 ## Unreleased
 #### Features
 #### Bug fixes and improvements
+- Fixed route parsing. Annotation duration from route is using to get segment's duration. [#6237](https://github.com/mapbox/mapbox-navigation-android/pull/6237)
```

Comment:
This doesn't tell me what exactly has been fixed, could we refactor this message?